recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - cython
     - numpy
     - libgdal
+    - openssl
```

Comment:
We don't want to merge with this but if this fails we know that the stack is unsolvable at the moment, if it passes then we are between a rock and a hard place b/c can can solve either way and we don't really know what to do to force the right `libgdal` with the latest `openssl` here :-(

Reply:
OK. This exposes the problem on Windows and shows that latest openssl is solvable on OS X. Let me try something else now. Hold on...